src/access/BUILD.gn

```diff
@@ -36,5 +36,6 @@ static_library("access") {
     "${chip_root}/src/lib/core",
     "${chip_root}/src/lib/support",
     "${chip_root}/src/platform",
+    "${chip_root}/src/protocols:im_status",  # For protocol error codes
```

Comment:
AccessControl module is internal business logic. It should not depend on src/protocols

Reply:
Updated logic. Hopefully cleaner now.